onnxruntime/test/server/unit_tests/http_routes_tests.cc

```diff
@@ -10,6 +10,7 @@ namespace onnxruntime {
 namespace server {
 namespace test {
 
+static const std::string predict_regex = R"(/v1/models/([^/:]+)(?:/versions/(\d+))?:(classify|regress|predict))";
```

Comment:
👍

Reply:
looks different from what is registered though?
```
/(?:v1/models/([^/:]+)(?:/versions/(\d+))?:(classify|regress|predict))|score
```